internal/server/kong/ws/config/compat/compatibility_table.go

```diff
@@ -834,6 +842,20 @@ var (
 				},
 			},
 		},
+		{
+			Metadata: config.ChangeMetadata{
+				ID:          config.ChangeID("P135"),
+				Severity:    config.ChangeSeverityError,
+				Description: standardCoreEntityMessage("vault", "3.0.0"),
```

Comment:
```suggestion
				Description: standardCoreEntityMessage("vault", "3.0"),
```

It seems we are consistently using `3.0` for update messages in this file.

Reply:
Updated this.